models/audit/index.js

```diff
@@ -0,0 +1,7 @@
+const audit = require('./audit.js');
+
+// export classes
+module.exports = {
+    Adudit: audit.Adudit,
```

Comment:
I think you misspelt this?

Reply:
indeed I did, although I sweat I fixed it.
and it says that your comment is outdated.

Reply:
ya this was fixed I think you reviewed an old version of the code